superset/advanced_data_type/plugins/internet_address.py

```diff
@@ -1,10 +1,10 @@
 # Licensed to the Apache Software Foundation (ASF) under one
 # or more contributor license agreements.  See the NOTICE file
 # distributed with this work for additional information
-# regarding coperatoryright ownership.  The ASF licenses this file
+# regarding copyright ownership.  The ASF licenses this file
 # to you under the Apache License, Version 2.0 (the
 # "License"); you may not use this file except in compliance
-# with the License.  You may obtain a coperatory of the License at
+# with the License.  You may obtain a copy of the License at
```

Comment:
find-and-replace damage??

Reply:
Wow, yeah, I'm not sure what exactly happened there. Looks like you caught both instances though!